remoteappmanager/static/js/home/configurables.js

```diff
@@ -1,85 +1,59 @@
-       
-        self.view = function (index) {
-            // Creates the View to add to the application entry.
-            var widget = $(view_template({
-                options: self.resolution_options,
-                index: index
-            }));
-            
-            widget.find("select").change(function() {
-                if (this.selectedIndex) {
-                    self.resolution = this.options[this.selectedIndex].value;
+        template:
+            '<div class="form-group">' +
+            '  <label>Resolution</label>' +
+            '  <select class="form-control" v-model="selected_value">' +
+            '    <option v-for="resolution_option in resolution_options">' +
+            '      {{resolution_option}}' +
+            '    </option>' +
+            '  </select>' +
+            '</div>',
+
+        data: function() {
+            return {
+                selected_value: this.value,
+                resolution_options: ['Window', '1920x1080', '1280x1024', '1280x800', '1024x768']
+            };
+        },
+
+        watch: {
+            value: function() {this.selected_value = this.value;}, // model -> view update
+            selected_value: function() {this.$emit('update:value', this.selected_value);} // view -> model update
+        },
+
+        methods: {
+            as_config_dict: function() {
+                var resolution = this.selected_value;
+
+                if (this.selected_value === 'Window') {
+                    var max_size = utils.max_iframe_size();
+                    resolution = max_size[0] + 'x' + max_size[1];
                 }
-            });
-            
-            return widget;
-        };
-    };
-    
-    ResolutionModel.prototype.tag = "resolution";
-   
-    ResolutionModel.prototype.as_config_dict = function() {
-        // Returns the configuration dict to hand over to the API request.
-        // e.g. 
-        // {
-        //    "resolution" : "1024x768"
-        // }
-        // The returned dictionary must be added to the configurable
-        // parameter under the key given by the tag member. 
-        var resolution = this.resolution;
-        
-        if (resolution === 'Window') {
-            var max_size = utils.max_iframe_size();
-            resolution = max_size[0]+"x"+max_size[1];
-        }
-        
-        return {
-            "resolution": resolution
-        };
-    };
-   
-    // Define all your configurables here.
-    var configurables = {
-        ResolutionModel: ResolutionModel
-    };
-    
-    var from_tag = function (tag) {
-        // Given a tag, lookup the appropriate configurable and
-        // return it. If the tag matches no configurable, returns null
-        for (var conf in configurables) {
-            if (configurables[conf].prototype.tag === tag) {
-                return configurables[conf];
+
+                return { resolution: { resolution: resolution } };
```

Comment:
This config dict is really weird, maybe we should change the representation needed by the container ?